src/services/account/services/SupportJWTService.js

```diff
@@ -154,7 +146,13 @@ class SupportJWTService {
 			const requestedUserId = userId.toString();
 			const currentUserId = params.account.userId.toString();
 
-			const jwt = await this.jwt.create(userId, currentUserId);
+			const account = await accountModel.findOne({ userId }).select('_id').lean().exec();
+			if (!account && !account._id) {
+				throw new Error(`Account for user with the id ${userId} does not exist.`);
+			}
+
+			const userData = await this.app.service('authentication').getUserData(requestedUserId, account._id);
+			const jwt = await this.jwt.create(currentUserId, userData);
 
 			await this.addToWhitelist(jwt);
```

Comment:
can we replace this with calling https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/src/services/authentication/logic/whitelist.js#L73 however we must generate the redisIdentifier from jwt before. maybe we can add an export in that file accepting the jwt token directly and replace all usages which are calling https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/src/services/authentication/logic/whitelist.js#L27 too.